engine/src/gldrv/gl_globals.h

```diff
@@ -315,8 +315,8 @@ typedef struct
     int  max_texture_dimension;
     int  max_movie_dimension;
     int  max_rect_dimension;
-    int  max_array_indices;
-    int  max_array_vertices;
+    unsigned int  max_array_indices;
+    unsigned int  max_array_vertices;
```

Comment:
should these be unsigned? It might be useful to use a `-1` for certain cases (like an invalid reference).
Please verify.

Reply:
This is to solve #245, where max_array_indices and max_array_vertices are compared to unsigned ints vertices_per_particle and vertices_per_particle in lines 26 to 49 of gfx/particle.cpp
They are only checked to be larger than 0, then used.